pyat/at/physics/linear.py

```diff
@@ -1142,7 +1142,7 @@ def get_strength(elem):
         try:
             k = elem.PolynomB[1]
         except (AttributeError, IndexError):
-            k = numpy.finfo(numpy.float64).eps
+            k = 1e-7
         return k
```

Comment:
I would rather set:
```python
try:
    k = elem.PolynomB[1]
    if k < 1.e-7:
        k = 0.0
except (AttributeError, IndexError):
    k = 0.0
return k
```
to avoid having drifts having a small k and using `betafoc`

Reply:
sure, sorry I totally miss-understood your comment.